metatensor-core/CMakeLists.txt

```
@@ -408,7 +413,14 @@ write_basic_package_version_file(
     COMPATIBILITY SameMinorVersion
 )
 
-install(FILES ${METATENSOR_HEADERS} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+install(FILES ${METATENSOR_HEADERS} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/metatensor)
```

This is a breaking change, it would require `#include <metatensor/metatensor.hpp>` by users, which I don't think we want to do.

```suggestion
install(FILES ${METATENSOR_HEADERS} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
```